### PR TITLE
CS: Apply ternary_to_null_coalescing fixer

### PR DESCRIPTION
**Pull request by derrabus**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | N/A
| License       | MIT
| Doc PR        | N/A

Since we don't need to support PHP 5 anymore, we can upgrade our older code to use the null coalescing operator and enforce its usage through our PHP CS Fixer configuration. I think, the usage of that operator improves the readability of our codebase at lot.

Note that the changes in this PR are (with the exception of `.php_cs.dist`) completely automated. You should get the same results by running the following command (and reverting the changes to PhpUnitBridge afterwards).

```sh
php-cs-fixer fix --rules='{"ternary_to_null_coalescing":true}' src
```

### CODE REVIEW

File: .php_cs.dist
Comment (OskarStark) [inline .php_cs.dist:20]: This should then be added to the @Symfony ruleset of pho-cs-fixer, too

**Review by OskarStark — APPROVED** [review]

**Review by nicolas-grekas — APPROVED** [review]
I'll let you merge up to 5.x right after you merge the PR :)

**Comment by derrabus** [comment]
The branches are in sync at the moment. Looks like a good moment to merge.

File: .php_cs.dist
Comment (keradus) [reply .php_cs.dist:20]: folks, do not hesitate to raise PR for Fixer repo ;)

**Pull request merged**
